app/models/comment.rb

```diff
@@ -2,14 +2,13 @@ class Comment < ApplicationRecord
   belongs_to :debate
   belongs_to :slack_user, class_name: Slack::User, foreign_key: :slack_user_id, optional: true
   validates :content, presence: true
+  delegate :image_url, :name, to: :user, prefix: true, allow_nil: true
```

Comment:
you should not allow nil because if there is no slack_user there will be anonymous user